<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="hw_btn">
       <property name="text">
        <string>手写</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lint_btn">
       <property name="text">
        <string>直线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cubic_btn">
       <property name="text">
        <string>曲线</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rect_btn">
       <property name="text">
        <string>矩形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="roundrect_btn">
       <property name="text">
        <string>圆角矩形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ellipse_btn">
       <property name="text">
        <string>椭圆</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="polygon_btn">
       <property name="text">
        <string>多边形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="triangle_btn">
       <property name="text">
        <string>三角形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pie_btn">
       <property name="text">
        <string>扇形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="arc_btn">
       <property name="text">
        <string>弧</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chord_btn">
       <property name="text">
        <string>弦</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="text_btn">
       <property name="text">
        <string>文本</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>图片</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="earse_btn">
       <property name="text">
        <string>橡皮擦</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="background_btn">
       <property name="text">
        <string>背景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pen_color_btn">
       <property name="text">
        <string>笔色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tkness_lb">
       <property name="text">
        <string>粗细</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="tkness_sld">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="state_lb">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 133, 26);</string>
       </property>
       <property name="text">
        <string>手写</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="screenshot_btn">
       <property name="text">
        <string>截图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="media_btn">
       <property name="text">
        <string>多媒体</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clear_btn">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="handwrite_grv"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
